[esx]/esx_identity/server/main.lua

```diff
@@ -373,27 +373,27 @@ else
 
 	ESX.RegisterServerCallback('esx_identity:registerIdentity', function(source, cb, data)
 		local xPlayer = ESX.GetPlayerFromId(source)
-		if xPlayer then
-			if not checkNameFormat(data.firstname) then
-            			xPlayer.showNotification(TranslateCap('invalid_firstname_format'), "error")
-            			return cb(false)
-        		end
-        		if not checkNameFormat(data.lastname) then
-            			xPlayer.showNotification(TranslateCap('invalid_lastname_format'), "error")
-            			return cb(false)
-        		end
-        		if not checkSexFormat(data.sex) then
-            			xPlayer.showNotification(TranslateCap('invalid_sex_format'), "error")
-            			return cb(false)
-        		end
-        		if not checkDOBFormat(data.dateofbirth) then
-            			xPlayer.showNotification(TranslateCap('invalid_dob_format'), "error")
-            			return cb(false)
-        		end
-        		if not checkHeightFormat(data.height) then
-            			xPlayer.showNotification(TranslateCap('invalid_height_format'), "error")
-            			return cb(false)
-        		end	
+		if not checkNameFormat(data.firstname) then
+	            	TriggerClientEvent('esx:showNotification',source,(TranslateCap('invalid_firstname_format'), "error")
```

Comment:
Issue with parenthesis for this TriggerClientEvent() function, please consider to update.

Reply:
fixed